frontend/components/Stepper/InitStepper.tsx

```diff
@@ -434,9 +435,18 @@ const ImportDataContent = (props: {
   const handleCreateClick = async () => {
     try {
       setIsLoading(true);
+      const requestBody: { recreate?: boolean } = {};
+      is409 && (requestBody.recreate = true);
```

Comment:
I think it is better to add a checkbox to let the user determine if recreate the job.